src/styles/style.css

```diff
@@ -0,0 +1,126 @@
+:root {
+  --main-color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+/* #region font */
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: bold;
+}
+
+/* #endregion */
+
+/* #region page */
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+/* #endregion */
+
+/* #region form */
+.search-form {
+  margin-top: 20px;
+}
+
+/* #endregion */
+
+/* #region input */
+.search-form__input {
+  font-family: inherit;
```

Comment:
The `font-family` property should not include 'inherit'. It should only specify the font name to avoid unexpected behavior.